src/assets/styles/10-settings/_typography.scss

```diff
@@ -46,6 +46,9 @@
   --heading-major-letter-spacing: -0.5px;
   --heading-minor-letter-spacing: 0.2px;
 
+  // font weight
+  --font-weight: 500;
```

Comment:
`--heading-font-weight` gives better context (if I have the context correctly?)

Reply:
I was thinking about similar variable name, but then realised we use this font-weight in various components, so wanted to make it more general.

Reply:
Fair enough, but I think it should have more context in the name if it's not used globally. I know it's associated with Helvetica but can you list the contexts it is used in?

Reply:
yes, the font-weight: 500 is used on all Helvetica headings.

I found that it is used for:

- cc-page-title__meta-label,
- cc-inpage-nav__title,
- th,
- c-hero-statement__title.

Looking at it again, I think it would be better to rename it to `--heading-font-weight`

Reply:
Yes I think you're right

HUZZAH!